build/package/auto-release-darwin-arm64.sh

```diff
@@ -10,6 +10,16 @@ plugin_dir=.devstream
 GOOS=$(go env GOOS)
 GOARCH=$(go env GOARCH)
 
+if [ ! $tag ] || [ ! $user ] || [ ! $repo ] || [ ! $github_token ] || [ ! $plugin_dir ]; then
+  echo "The following variables cannot be empty!"
+  echo "tag="$tag
+  echo "user="$user
+  echo "repo="$repo
+  echo "github_token="$github_token
```

Comment:
if github_token isn't empty but maybe for example, tag is empty, github_token will be printed out on the screen, which isn't safe.

Reply:
+1

Reply:
1.  add "set -o nounset" for undefined parameter
2. do not  print the true token